test/value_parser_test.rb

```diff
@@ -85,17 +85,20 @@ class FilterTest < ActiveSupport::TestCase
     assert_equal result.max, end_time.to_s
   end
 
-  test "parses range for Date string range" do
-    options = {
-      supports_ranges: true
-    }
-    start_date = Date.new(2018, 1, 26)
-    end_date = Time.new(2018, 1, 29)
-    range_string = "#{start_date}...#{end_date}"
-    parser = Sift::ValueParser.new(value: range_string, options: options)
-
-    result = parser.parse
-    assert_instance_of Range, result
-    assert_equal result.max, end_date.to_s
+  [:date, :time, :datetime].each do |type|
+    test "parses range for Date string range and normalizes Date values for type #{type}" do
+      options = {
+        supports_ranges: true
+      }
+
+      start_date = "2018-01-01T10:00:00Z[Etc/UTC]"
```

Comment:
I would expect the input strings to be different for `date`, `time` and `datetime` examples. I think the tests should verify that the expected input formats get parsed correctly. I think it would be nice to have an invalid example as well and verify that gets handled as expected.